torch/testing/_internal/common_methods_invocations.py

```diff
@@ -785,7 +785,7 @@ def sample_inputs_linalg_pinv_hermitian(op_info, device, dtype, requires_grad=Fa
         o.kwargs = {"hermitian": True}
     return out
 
-def sample_inputs_linalg_solve(op_info, device, dtype, requires_grad=False):
+def sample_inputs_linalg_solve(op_info, device, dtype, requires_grad=False, vector_rhs_allowed=True):
```

Comment:
Extend the docstring to describe vector_rhs_allowed, too, and why it's here (and why it may go away in the future).